webpack/components/TabbedNarrative.jsx

```diff
@@ -13,11 +13,40 @@ class TabbedNarrative extends React.Component {
       titles: null
     };
 
+    const hookupMouseovers = function hookupMouseovers(container) {
+      container
+        .querySelectorAll("[data-highlight-dans]")
+        .forEach(targetElem => {
+          targetElem.addEventListener("mouseover", ({ target }) => {
+            const sectionIndices = target.attributes[
+              "data-highlight-dans"
+            ].value.split(",");
+            sectionIndices.forEach(index => {
+              container
+                .querySelector(`[data-index="${index}"]`)
```

Comment:
On L2, I in order to get the highlighting to work when the shōdan map is expanded, I had to change this to
```
                .querySelectorAll(`[data-index="${index}"]`).forEach(targetElem => {
                  targetElem.classList.add("shodan-map__item--highlight");
              });
```
Same for the class `remove` below. I think this is because there are actually two copies of the section map on L2 -- one expanded, one compressed, so without the `forEach()` it only finds the section blocks on one of them.

Reply:
Ha -- great spot, thanks!  I hadn't really been thinking about this on L2, tbh, but I should have done as the code  work there just as well, except for the issue you found.

> I think this is because there are actually two copies of the section map on L2

Yep.  There are two literally 100% identical elements stuffed into the DOM.  This is... unexpected, but (short of removing the duplicate element) is probably best worked-around with exactly the fix you suggest.  I'll get it committed ASAP.